web_src/js/features/repo-issue.js

```diff
@@ -287,6 +287,20 @@ export function initRepoPullRequestMergeInstruction() {
   });
 }
 
+export function initRepoPullRequestAllowEditsFromMaintainers() {
+  $('#allow-edits-from-maintainers input').on('change', function () {
+    let url = '';
+    if ($(this).is(':checked')) { // allow edits
+      url = `${window.location}/allow_edits`;
+    } else { // disallow edits
+      url = `${window.location}/disallow_edits`;
+    }
+    $.post(url, {
+      _csrf: csrfToken
+    });
```

Comment:
I think it may be better to watch for a `input` event on the checkbox, send the ajax and on success alter the checked state. That way, the user has visual confirmation that the request was accepted by the server. Not a major issue thought.

Reply:
Yes, I tried to implement this but I wasn't  able to prevent the change of the checkbox - will try it with `input`

Reply:
On the other hand, I'm note sure it's ideal to have this "delay" when clicking the checkbox, it might confuse the user. So current method of just asynchronously sending it is probably fine, at least if we expect this request to never fail.

Reply:
My idea was to have a loading indicator that replaces the checkbox until loading is finished and then display the chrckbox in the correct state, but I don't know how I could implement that

Reply:
You could try toggling the `is-loading` class defined below which should show a spinner. But I'm not sure it will work correctly for such small elements.

https://github.com/go-gitea/gitea/blob/4bbbf356547557bd6b36d1d6cb73540a5febf120/web_src/less/animations.less#L6

Reply:
I added a failed popup in f086784 and will try to add a loading indicator.